cmd/snap/cmd_snap_op.go

```diff
@@ -257,7 +280,7 @@ func (x *cmdInstall) Execute([]string) error {
 		name = snapName
 	}
 
-	return listSnaps([]string{name})
+	return showDone([]string{name}, "installed")
```

Comment:
If you follow my advice above please also switch this to use vanilla operation (``"install"``)